week-07/FoxClub/day-05/Services/FoodService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using day05.Interfaces;
+
+namespace day05.Services
+{
+    public class FoodService : IFood
+    {
+        public List<string> AllFoods { get; private set; }
+        public List<string> AllDrinks { get; private set; }
+
+        public FoodService()
+        {
+            AllDrinks = new List<string>()
+            {
+                "Lemonade","Coca Cola", "Alcohol", "Piss"
+            };
+            AllFoods = new List<string>()
+            {
+                "Pizza", "Donuts", "Spaghetti"
+            };
+        }
+
+        public void AddFood(string food)
+        {
+            throw new NotImplementedException();
```

Comment:
✂️

Reply:
What does this mean? is it because i have yet to implement the method body?